scipy/io/netcdf.py

```diff
@@ -987,6 +983,50 @@ def __setitem__(self, index, data):
                 self.data.resize(shape)
         self.data[index] = data
 
+    def _get_missing_value(self):
+        """
+        Returns the value denoting "no data" for this variable.
+
+        If this variable does not have a missing/fill value, returns None.
+        """
+
+        if ('_FillValue' in self._attributes):
```

Comment:
Code style issue: no parenthesis in if statements
`if '_FillValue' in self._attributes:`